### PR TITLE
add Rankine scale

### PR DESCRIPTION
**Pull request by lance-pyles**
black formatting

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
Cool!  Python's builtin [__round()__](https://docs.python.org/3/library/functions.html#round) function takes an optional parameter `ndigits`.  Should these functions each have an optional `ndigits` parameter that defaults to 2 but if the caller sets it then that values is passed down to the round call?  This gives the caller more flexibility to set the precision if that helps in their use case.  Thoughts?

File: conversions/temperature_conversions.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 """ Convert between different units of temperature """
 
 
-def celsius_to_fahrenheit(celsius: float) -> float:
+def celsius_to_fahrenheit(celsius: float, digits: int = 2) -> float:
```
Comment (cclauss) [inline conversions/temperature_conversions.py:4]: The `round()` parameter is `ndigits`.

Comment (cclauss) [reply conversions/temperature_conversions.py:4]: Please also add a test or two to prove that `ndigits` is respected.

Comment (lance-pyles) [reply conversions/temperature_conversions.py]: fixed

Comment (lance-pyles) [reply conversions/temperature_conversions.py]: tests coming in a new PR

**Pull request merged**